proposals/csharp-7.0/tuples.md

```diff
@@ -0,0 +1,437 @@
+# Tuples 
+
+This proposal specifies the changes required to the [C# 6.0 (draft) Language specification](../../spec/introduction.md) to support *Tuples* as a new [value type](../../spec/types.md#value-types).
+
+## Changes to [Lexical structure](../../spec/lexical-structure.md)
+
+> The grammar for [Literals](../../spec/lexical-structure.md#literals) is extended to include `tuple_literal`:
```

Comment:
>lexical-structure.md [](start = 40, length = 20)

Tuple literals are not a new lexical structure.  They belong in a new section in expressions, and require semantics to be defined there.

Reply:
logged in #3186